### PR TITLE
composer: bump up to support Laravel 9

### PR DESCRIPTION
**Pull request by StevenSteel**

### CODE REVIEW

**Review by jorenvh — CHANGES_REQUESTED** [review]
@StevenSteel do you mind making a few extra changes:

- update the Github actions to also run for Laravel 9, php 8 & php 8.1
- Drop php 7.4 support
- Add php 8.1 support

**Comment by jorenvh** [comment]
@StevenSteel Laravel 8 should still be supported, run the tests against both Laravel 8 & 9

**Review by jorenvh — CHANGES_REQUESTED** [review]
Fix failing tests and run against both Laravel 8 & Laravel 9

File: .php-cs-fixer.cache [hunk 1]
```diff
@@ -0,0 +1 @@
..."ordered_imports":{"sort_algorithm":"alpha"},"no_unused_imports":true,"not_operator_with_successor_space":true,"trailing_comma_in_multiline":true,"phpdoc_scalar":true,"unary_operator_spaces":true,"binary_operator_spaces":true,"blank_line_before_statement":{"statements":["break","continue","declare","return","throw","try"]},"phpdoc_single_line_var_spacing":true,"phpdoc_var_without_name":true,"class_attributes_separation":{"elements":{"method":"one"}},"single_trait_insert_per_statement":true},"hashes":{"src\/Actions\/ActionInterface.php":3794870253,"src\/Actions\/ClearCacheAction.php":3391253236,"src\/LaravelStoryblokFacade.php":465150087,"src\/Http\/Controllers\/StoryblokWebhookController.php":936241491,"src\/Http\/Middleware\/VerifyStoryblokWebhookSignature.php":2251817492,"src\/Commands\/ClearStoryblokCacheCommand.php":452607207,"src\/LaravelStoryblokServiceProvider.php":2359557763,"src\/Storyblok.php":3594343834,"tests\/StoryblokTest.php":1140628678,"tests\/Http\/Controllers\/StoryblokWebhookControllerTest.php":2828080726,"tests\/Commands\/ClearStoryblokCacheCommandTest.php":2242150547,"tests\/Stubs\/WebhookActionWithoutInterface.php":1709632369,"tests\/TestCase.php":3588041990}}
```
Comment (jorenvh) [inline .php-cs-fixer.cache]: This file should not be in git, add it to gitignore

**Review by jorenvh — CHANGES_REQUESTED** [review]

File: .php-cs-fixer.cache (hunk 1 above)
Comment (StevenSteel) [reply .php-cs-fixer.cache]: done

**Review by jorenvh — APPROVED** [review]

**Pull request merged**